masonry/src/properties/types/length.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright 2025 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use masonry_core::debug_panic;
+
+/// A value representing a width, height, or similar distance value.
+///
+/// Its value is always finite and non-negative.
+#[derive(Clone, Copy, PartialEq)]
+pub struct Length {
+    value: f64,
+}
+
+impl std::fmt::Debug for Length {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", self)
```

Comment:
```suggestion
        write!(f, "{}px", self)
```
I don't think there's any advantage to this terseness here.

Reply:
It's not terse, the debug impl re-uses the Display impl. This is to avoid code duplication.

Reply:
Oops... Thanks!